src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkAddressChange.UnknownUnix.cs

```diff
@@ -6,10 +6,19 @@ namespace System.Net.NetworkInformation
 {
     public partial class NetworkChange
     {
+        // Introduced for supporting design-time loading of System.Windows.dll
+       [Obsolete("This API supports the .NET Framework infrastructure and is not intended to be used directly from your code.", true)]
+       public static void RegisterNetworkChange(NetworkChange nc) { }
```

Comment:
Even though this is marked obsolete - should we still throw PNSE in the body?

Reply:
It's a nop in corefx on all of the other platforms, e.g.
https://github.com/dotnet/corefx/blob/60003797de586098f83eea0719d31787b9c2e141/src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkAddressChange.Windows.cs#L16-L17

In fact, it has the same nop implementation on all platforms, so the stub for it should just be moved to the platform-neutral file:
https://github.com/dotnet/corefx/blob/master/src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkAddressChange.cs